docs/index.md

```diff
@@ -1,11 +1,11 @@
 ---
 layout: default
-description: "Service Mesh Performance Specification (SMPS): Measure your service mesh performance using SMPS. 
-SMPS provides a universal performance index to gauge your mesh’s efficiency against deployments in other organizations’ environments."
+description: "Service Mesh Performance (SMP): Measure your service mesh performance using SMP. 
+SMP provides a universal performance index to gauge your mesh’s efficiency against deployments in other organizations’ environments."
 author: Lee Calcote
-twitter: smp_spec
+twitter: smp
```

Comment:
```suggestion
twitter: smp_spec
```
The twitter handle is @smp_spec.

Reply:
@chandrashritii, @PluckyPrecious suggested the change, I thought of including it as it would be adapted in the future. Should I revert back?

Reply:
> @chandrashritii, @PluckyPrecious suggested the change, I thought of including it as it would be adapted in the future. Should I revert back?

@JubayerJoy kindly revert. Apologies for the inconvenience caused.

Reply:
@PluckyPrecious reverted back. No worries! 😄